deprecations/README.md

```diff
@@ -24,6 +24,7 @@ The table columns have the following meanings:
 
 | Feature | Comments | Deprecated | Discontinued |
 | :-- | :-- | :-- | :-- |
+| `releasedDate` in [`Dates`](../channel-manager-operations/availabilityBlock.md#dates) object in [Availability block](../channel-manager-operations/availabilityBlock.md#availability-block) | Replaced by `fixedDate` in [`Release strategy`](../mews-operations/availabilityBlock.md#release-strategy) of  [`Availability block`](../channel-manager-operations/availabilityBlock.md#availability-block) | 10 Oct 2024 | - |
```

Comment:
Somehow we need to say that this is in the request for [CHM: Process availability block](../channel-manager-operations/availabilityBlock.md#process-availability-block).